buildout.cfg

```diff
@@ -26,7 +26,10 @@ eggs =
 # development egg, you really need to un-pin it here.
 lizard-auth-server =
 Django = 1.6.6
+PyJWT = 1.4.0
```

Comment:
Het is meer dat de eerste lijst echt expliciet gekozen versies zijn en dat je de tweede lijst in principe kan deleten en de nieuwste versies erneer kan zetten.

Zo had ik het onderscheid in ieder geval bedoeld toen ik het in nensskel zette.

Dus als er iets in de eerste lijst staat is het omdat er een nieuwere/andere versie is waar problemen mee zijn (meestal).